Dockerfile

```diff
@@ -1,3 +1,17 @@
+FROM golang:alpine AS build
+
+# Set the working directory
+WORKDIR /go/src/github.com/adnanh/webhook
+ENV WEBHOOK_VERSION=2.8.1
+
+RUN apk add --update -t build-deps curl libc-dev gcc libgcc
+
+RUN curl -L --silent -o webhook.tar.gz https://github.com/adnanh/webhook/archive/${WEBHOOK_VERSION}.tar.gz && \
```

Comment:
This neither a good nor secure way to install software and doesnt follow the Ansible way.